docusaurus.config.js

```diff
@@ -322,6 +322,30 @@ module.exports = {
         versions: VERSIONS_JSON,
       },
     ],
+    function (context, options) {
+      return {
+        name: 'beamer',
+        injectHtmlTags({content}) {
+          // Only inject Beamer when deployed to production, so we don't use up Beamer "users"
+          if (process.env.NODE_ENV === "production") {
```

Comment:
I think I did this correctly. When i run locally (ie localhost:3000), I don't see Beamer installed as expected. But I do see it on this PR preview 👍